src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -120,8 +120,8 @@
   <data name="NavigationToolbarNewWindow.Label" xml:space="preserve">
     <value>New window</value>
   </data>
-  <data name="CopyLocation" xml:space="preserve">
-    <value>Copy location</value>
+  <data name="CopyAsPath" xml:space="preserve">
+    <value>Copy as path</value>
```

Comment:
```suggestion
    <value>Copy path</value>
```

Reply:
File Explorer labels it as "Copy as path". Do we not need to match the label?
![image](https://github.com/files-community/Files/assets/66369541/cead798a-79c2-48c7-904e-7c900dbdcb9e)

Reply:
It doesn't have to be the same, I'm not sure what the "as" is for.

Reply:
I changed it to "Copy path". Actually, I think this is easier to understand too.